.github/workflows/trivy.yml

```diff
@@ -0,0 +1,27 @@
+name: Trivy
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+
+    - name: Build the Docker image
+      run: docker build . -t appwrite_image:latest
```

Comment:
Let's build like how we do in tests to make use of caching: 

https://github.com/appwrite/appwrite/blob/fce18647d2352e4cfce7ce75fd103a21bafbf7d0/.github/workflows/tests.yml#L28-L44